### PR TITLE
EDUCATOR-786 | Support Django 1.11

### PR DESCRIPTION
**Pull request by iloveagent57**
Upgrade django_nose; Mock out read_replica for testing (see https://code.djangoproject.com/ticket/23718)

Reviewers:
- [ ] @bmedx 
- [ ] @doctoryes 

FYI Reviewers:
@edx/educator-all

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = django{18,19,110}
+envlist = django{18,19,110,111}
```
Comment (efischer19) [inline tox.ini]: 1.9 is [no longer a supported release](https://www.djangoproject.com/download/), should we continue to test against it?

File: test-requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ddt==0.8.0
-django-nose==1.4.1
+django-nose==1.4.4
```
Comment (efischer19) [inline test-requirements.txt]: Could you update the tox requirement in here to the latest version (2.7.0) as well? I was having some issues with that in my ora work.

File: tox.ini (hunk 1 above)
Comment (iloveagent57) [inline tox.ini]: Oh, good point.

File: .travis.yml [hunk 3]
```diff
@@ -20,3 +19,11 @@ script:
   - "tox"
 after_success:
   coveralls
+deploy:
+  provider: pypi
+  user: edx
+  distributions: sdist bdist_wheel
+  skip_upload_docs: true
+  password:
+    secure: the-encrypted-password
```
Comment (efischer19) [inline .travis.yml]: How exactly does this work? Is the password stored in travis somewhere?

Comment (doctoryes) [inline .travis.yml]: @efischer19 Does this page answer your question?:
https://openedx.atlassian.net/wiki/display/OpenOPS/Publishing+a+Package+to+PyPI+using+Travis

Comment (efischer19) [inline .travis.yml]: Sure does, thanks!

**Pull request merged**